extensions/amp-story/amp-story-analytics.md

```diff
@@ -2,6 +2,31 @@
 
 This document details triggers associated with AMP Stories. If you're looking for a guide to setting up analytics for your AMP pages, see [this blog post](https://blog.amp.dev/2019/08/28/analytics-for-your-amp-stories/)
 
+## Shopping triggers
+
+### story_shopping_plp_view
```

Comment:
Please follow the same header convention as the other triggers.
`Tooltip click through trigger ("on": "story-click-through")`